_posts/2021-04-21-frequency-band-control-in-unetstack.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+comments: true
+title: Frequency band control using UnetStack
+date: 21/04/2021
+author: Shiraz S
+categories: howto
+feature-img: "assets/img/freqBandControl/title.png"
+thumbnail: "assets/img/freqBandControl/title.png"
+tags: [howto, unetstack, modems, phy, frequency, band, fdma]
+---
+
+## The basics
+
+An underwater acoustic modem essentially sends out sound signals underwater to transmit information. The typical transmit processing takes in user's binary data and after error correction encoding, modulates a carrier according to some modulation scheme such as OFDM or FHBFSK, the details of which the reader can understand from other sources. 
+
+For standard medium frequency Subnero modems, the fundamental centre frequency `C=24kHz`.
```

Comment:
It might make sense to not just focus on a Subnero modem, but perhaps specify what it is for Unet audio and also an example of a UnetStack-based modem from Subnero (and link it to the products page)

Reply:
changed text. need to add reference

Reply:
Would be good if Manu can add the correct product related references for this and related comments